modal/cli/run.py

```diff
@@ -63,6 +67,16 @@ def _add_click_options(func, signature: inspect.Signature):
     return func
 
 
+def _get_clean_stub_description(func_ref: str) -> str:
```

Comment:
Duplicate method from above?

Reply:
Yeh I just copied the fn rather than having it in one place and importing it. It's really simple so this seems fine? Rule of three and all that, I could centralize it then.